offer-ms/src/main/java/fr/polytech/ig5/mnm/offerms/DTO/OfferCreateDTO.java

```diff
@@ -0,0 +1,44 @@
+package fr.polytech.ig5.mnm.offerms.DTO;
+
+import fr.polytech.ig5.mnm.offerms.models.ContractType;
+import lombok.Data;
+import org.springframework.data.annotation.CreatedDate;
+
+import javax.validation.constraints.NotEmpty;
+import javax.validation.constraints.NotNull;
+import java.sql.Date;
+import java.time.Instant;
+import java.util.UUID;
+
+@Data
+public class OfferCreateDTO {
+
+    @NotEmpty(message = "an offer must have a description")
+    private String description;
+
+    private UUID companyId;
+
+    @CreatedDate
+    private Instant creationDate;
```

Comment:
Should use same type for dates
check https://stackoverflow.com/questions/32437550/whats-the-difference-between-instant-and-localdatetime
localdatetime should do the trick for our needs